ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/BlockPropagationManager.java

```diff
@@ -181,6 +165,31 @@ private void onBlockAdded(final BlockAddedEvent blockAddedEvent) {
     }
   }
 
+  private void maybeRetrieveNonAnnouncedBlocks(final Block newBlock) {
```

Comment:
method name has "retrieve" but returns void. Should this maybe be something like "process"?

Reply:
make sense, will change